scrapper.py

```diff
@@ -35,7 +49,8 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed in self.seed_urls:
+            response = requests.get(seed)
```

Comment:
this method should somehow extract the URL, I do not see it to this moment

Reply:
Should be done now